templates/raygun.rb

```diff
@@ -0,0 +1,7 @@
+Raygun.setup do |config|
+  config.api_key = ENV['RAYGUN_APIKEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.